src/components/Transaction.tsx

```diff
@@ -14,10 +15,14 @@ type TransactionProps = {
 
 export const Transaction: React.FunctionComponent<TransactionProps> = props => {
   const { t } = useTranslation();
+
+  const [activeClass, setActiveclass] = useState<boolean>(false);
```

Comment:
not needed for this PR, but FYI i suggest to check this repo: https://github.com/streamich/react-use which contains a lot of common case hooks like https://github.com/streamich/react-use/blob/master/docs/useToggle.md 

(or this one to track state history: https://github.com/streamich/react-use/blob/master/docs/useStateWithHistory.md)